scripts/clean_deploy.py

```
@@ -0,0 +1,236 @@
+    """Print colored message"""
+    try:
+        print(f"{color}{message}{NC}")
+    except UnicodeEncodeError:
+        message_ascii = message.encode('ascii', 'replace').decode('ascii')
+        print(f"{color}{message_ascii}{NC}")
+
+def get_neon_url():
+    """Load NEON_DB_URL from .env file"""
+    env_path = Path(__file__).parent.parent / '.env'
+
+    with open(env_path, 'r', encoding='utf-8') as f:
```

Ruff (0.14.10)

Unnecessary mode argument

Remove mode argument

(UP015)